subworkflows/local/amp.nf

```diff
@@ -28,9 +28,7 @@ workflow AMP {
             .map {
                 file ->
                     def meta  = [:]
-                    ext_index       = file.getName().lastIndexOf(".")
-                    ext             = file.getName().substring(ext_index)
-                    meta['id'] = ext == '.gz' ? file.getName() - '.hmm.gz' :  file.getName() - '.hmm'
+                    meta['id'] = file.extension == 'gz' ? file.name - '.hmm.gz' :  file.name - '.hmm'
```

Comment:
I was also thinking, should we assert that the extension is indeed `.hmm` or do we simply expect that?

Reply:
I believe it is almost always that - if you look at the hmmer database, and also databases like Pfam (which I assume is where most people get their stuff) it's always `.hmm`.

But I'm very much in the realm of dealing with tools I have no clue about now :laughing: